.claude/agents/ui-designer.md

```
@@ -0,0 +1,134 @@
+デザインレビュー時は、Playwright MCP を使用して実際のブラウザでUIを確認する。
+
+**前提条件:**
+
+- 開発サーバーはすでに起動している（`npm run dev` 不要）
+- アプリケーションURL: `http://localhost:3000`
+
+## 出力形式
+
+### 設計提案時
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)